requirements.txt

```diff
@@ -0,0 +1,7 @@
+appdirs==1.4.3
+Django==1.11
+packaging==16.8
+psycopg2==2.7.1
+pyparsing==2.2.0
+pytz==2017.2
+six==1.10.0
```

Comment:
Apart from Django and psycopg2 why do we need rest of them. BTW we do need a requirements.txt.